### ISSUE TITLE
Make more instruments

### ISSUE THREAD

**Issue by IsotoxalDev** [QuadCubedStudios/GoDAW#45]
GoDAW uses custom instruments instead of relying on Soundfonts and VST. Support for both Soundfonts and VST can be expected.

**Comment by oparisy** [comment]
Since you are referring to VST: my understanding of the current VST3 SDK licensing situation is that it will require this project to be GPLv3, while you chose an MIT license.

An alternate approach seems to be the use of a reverse engineered VST3 host API, whose legal status seems arguable (and all projects using those seem to be GPL anyway).

So I guess I'm curious about how you plan to navigate those waters?

Just to be clear let me say that I'm very sympathetic to the idea of a Godot VST host under a liberal license!

**Comment by Technohacker** [comment]
Oh that's interesting, VST3's SDK is open source? I was fully expecting it to be proprietary

Since we have very few authors in the current codebase, it's easy for us to change licenses if required, and I personally feel fine with licensing the codebase under GPL if required

That being said, VST integration isn't too high a priority for now, but that's a good point to consider. Thanks for the heads up!

**Comment by oparisy** [comment]
Yes, my understanding is that the VST3 SDK is dual licenced (commercial and
GPLv3).

Thanks for your answer!

Le lun. 30 mai 2022, 18:37, Kevin Kuriakose ***@***.***> a
écrit :

**Comment by IsotoxalDev** [comment]
To implement VST there are a few options as of now: 
1. Change License ( wouldn't be too hard, maybe.... )
2. Make VST a GPL licensed addon for GoDAW that would be available on an official store. Which wouldn't be bundled with the official build.
